### PR TITLE
markdownify

### PR DESCRIPTION
**Pull request by amshamah419**
## Status
WIP

## Related Issues
fixes: https://github.com/demisto/etc/issues/20559
depends on: https://github.com/demisto/dockerfiles/pull/1049

## Description
An automation to convert html to markdown

## Screenshots
image

## Required version of Demisto
x.x.x

## Does it break backward compatibility?
   - Yes
       - Further details:
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Dependencies
Mention the dependencies of the entity you changed as given from the precommit hooks in checkboxes, and tick after tested them.
- [ ] Dependency 1
- [ ] Dependency 2
- [ ] Dependency 3

## Additional changes
Describe additional changes done, for example adding a function to common server.

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](https://github.com/demisto/content/blob/tomd/Scripts/markdownify/markdownify.yml)
- [ ] [CHANGELOG](https://github.com/demisto/content/blob/tomd/Scripts/markdownify/CHANGELOG.md)

### CODE REVIEW

File: Scripts/HTMLtoMD/HTMLtoMD.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import demistomock as demisto
+from CommonServerPython import *
+from markdownify import markdownify as md
+
+
+def html_to_md_command(args):
```
Comment (bakatzir) [inline Scripts/HTMLtoMD/HTMLtoMD.py]: add type annotation for args and return

File: Scripts/HTMLtoMD/HTMLtoMD.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    html = str(args.get('html'))
```
Comment (bakatzir) [inline Scripts/HTMLtoMD/HTMLtoMD.py]: can you str() a None? please handle

File: Scripts/HTMLtoMD/HTMLtoMD.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    markdown = md(html)
+    result = {
+        "Original": str(html),
```
Comment (bakatzir) [inline Scripts/HTMLtoMD/HTMLtoMD.py]: you've already str() in line 7

File: Scripts/HTMLtoMD/HTMLtoMD.py (hunk 2 above)
Comment (amshamah419) [reply Scripts/HTMLtoMD/HTMLtoMD.py]: The argument is listed as mandatory so I set a default empty string if none is provided.

**Pull request merged**